Tasks/Maven/pmdformaven.ts

```diff
@@ -37,43 +39,52 @@ export function processPmdOutput(sourcesDirectory:string) : ar.AnalysisResult {
 
 // Verifies the existence of the HTML output file.
 // Modifies the relevant field within the returned object accordingly.
-// Task fails if the file cannot be found.
 function processPmdHtml(analysisResult:ar.AnalysisResult, path:string):ar.AnalysisResult {
-    // Task fails if the file cannot be found.
     if (!tl.exist(path)) {
-        throw new Error("Could not find PMD HTML output. Check that PMD ran successfully and that it is writing to the default location.");
+        tl.debug('PMD HTML not found at ' + path);
```

Comment:
No longer throws an error. See comment above about coding defensively in case module detection throws a false positive.